### PR TITLE
modified the Introduction and added few more points -- OLM #111

### PR DESCRIPTION
**Pull request by laxmikantbpandhare**
Reopens #111

### CODE REVIEW

File: content/en/docs/_index.md [hunk 1]
```diff
@@ -29,5 +29,58 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+
+## How OLM solves the cluster admin and developer needs.
+
+### Operator Installation and Management
+
+This packaging mechanism helps to simplify the multiple steps involved in deploying an Operator. OLM fulfills necessary metadata for visualizing them in compatible UIs, including installation instructions and API hints in the form of CRD descriptors. It helps cluster admin, developers to manage the lifecycle of all Operators and their associated services running across their clusters.
+
+### Dependency Resolution
+
+Using OLM eliminates the need for developers and cluster admins to have to worry about managing dependancies when developing one or more operators. OLM offers dependency resolution and upgrades the lifecycle of running operators. This functionality helps operator developers to resolve the problems related to dependency management.
```
Comment (dinhxuanvu) [inline content/en/docs/_index.md:41]: `dependencies`

File: content/en/docs/_index.md [hunk 2]
```diff
@@ -29,5 +29,58 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+
+### Works similar to package Managers
+
+Operator developers may need to use one or more APIs for the operator installation and OLM helps solve this use case by verifying that all of the required APIs are available for the operator. If they are not available, then OLM will not install the operator.
```
Comment (dinhxuanvu) [inline content/en/docs/_index.md:45]: OLM will attempt to find an operator that owns those missing requiredAPIs and install it. If there is no operator satisfying requiredAPIs, then OLM won't install the main operator.

File: content/en/docs/_index.md [hunk 3]
```diff
@@ -29,5 +29,58 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+* Defines the set of namespaces that OLM may grant namespaced permissions in.
+
+## Building blocks of OLM
+
+### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)
+A `ClusterServiceVersion (CSV)` represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. 
+
+
+### [CatalogSource](/docs/concepts/crds/catalogsource/)
+A `CatalogSource` represents a store of metadata that OLM can query to discover and install operators and their dependencies.
```
Comment (dinhxuanvu) [inline content/en/docs/_index.md:61]: `operator bundles and metadata`

File: content/en/docs/_index.md [hunk 4]
```diff
@@ -29,5 +29,58 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+### [OperatorCondition](/docs/concepts/crds/operatorcondition/)
+An `OperatorCondition` is CustomResourceDefinition that creates a communication between OLM and an operator it manages. Operators may write to the `Status.Conditions` array to modify OLM management the operator.
+
+
+### [Subscriptions](/docs/concepts/crds/subscription)
+A `Subscription` represents an intention to install an operator. Subscriptions are Custom Resources that relate an operator to a CatalogSource. Subscriptions describe which [channel](/docs/glossary/#channel) of an operator package to subscribe to and whether to perform updates automatically or manually. If set to automatic, the Subscription ensures OLM will manage and upgrade the operator to ensure the latest version is always running in the cluster.
+
+
+### [InstallPlan](/docs/concepts/crds/installplan/)
+An `InstallPlan` defines a set of resources to be created in order to install or upgrade to a specific version of a ClusterService defined by a CSV.
```
Comment (dinhxuanvu) [inline content/en/docs/_index.md:73]: `a specific version of an operator`

File: content/en/docs/_index.md [hunk 5]
```diff
@@ -29,5 +29,58 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)
+A `ClusterServiceVersion (CSV)` represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. 
+
+
+### [CatalogSource](/docs/concepts/crds/catalogsource/)
+A `CatalogSource` represents a store of metadata that OLM can query to discover and install operators and their dependencies.
+
+
+### [OperatorCondition](/docs/concepts/crds/operatorcondition/)
+An `OperatorCondition` is CustomResourceDefinition that creates a communication between OLM and an operator it manages. Operators may write to the `Status.Conditions` array to modify OLM management the operator.
```
Comment (dinhxuanvu) [inline content/en/docs/_index.md:65]: `to inform OLM regarding operators' conditions that may change the way OLM manage the lifecycle of the operators.`

**Review by dinhxuanvu — COMMENTED** [review]
Thank you for your PR. Just a few comments/suggestions.

Can you please squash your commits?

File: content/en/docs/_index.md (hunk 1 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Corrected the above suggestion.

File: content/en/docs/_index.md (hunk 2 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Added the above description.

File: content/en/docs/_index.md (hunk 3 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Added it with the above catalog source description.

File: content/en/docs/_index.md (hunk 5 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Added the above description.

File: content/en/docs/_index.md (hunk 4 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified the description.

File: content/en/docs/_index.md [hunk 6]
```diff
@@ -29,5 +29,57 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
 ### Declarative UI controls
 Operators can behave like managed service providers. Their user interface on the command line are APIs. For graphical consoles OLM annotates those APIs with descriptors that drive the creation of rich interfaces and forms for users to interact with the Operator in a natural, cloud-like way. 
 
+
+## How OLM solves the cluster admin and developer needs.
```
Comment (timflannagan) [inline content/en/docs/_index.md:33]:
```suggestion
## How OLM solves the cluster admin and developer needs
```

File: content/en/docs/_index.md [hunk 7]
```diff
@@ -29,5 +29,57 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+
+### Operator Installation and Management
+
+This packaging mechanism helps to simplify the multiple steps involved in deploying an Operator. OLM fulfills necessary metadata for visualizing them in compatible UIs, including installation instructions and API hints in the form of CRD descriptors. It helps cluster admin, developers to manage the lifecycle of all Operators and their associated services running across their clusters.
```
Comment (timflannagan) [inline content/en/docs/_index.md:37]:
```suggestion
This packaging mechanism helps to simplify the multiple steps involved in deploying an Operator. OLM fulfills the necessary metadata for visualizing them in compatible UIs, including installation instructions and API hints in the form of CRD descriptors. It helps cluster admins and developers to manage the lifecycle of all Operators and their associated services running across their clusters.
```

File: content/en/docs/_index.md [hunk 8]
```diff
@@ -29,5 +29,57 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+
+### Dependency Resolution
+
+Using OLM eliminates the need for developers and cluster admins to have to worry about managing dependencies when developing one or more operators. OLM offers dependency resolution and upgrades the lifecycle of running operators. This functionality helps operator developers to resolve the problems related to dependency management.
```
Comment (timflannagan) [inline content/en/docs/_index.md:41]: I don't know if there's any resemblance on Operator vs. operator throughout the rest of the docs, but it looks like you had used Operator in previously sections, vs. operator in subsequent ones. I think we should aim towards some more consistency in that regard.

```suggestion
Using OLM eliminates the need for developers and cluster admins to have to worry about managing dependencies when developing one or more Operators. OLM offers dependency resolution and upgrades the lifecycle of running Operator. This functionality helps operator developers to resolve the problems related to dependency management.
```

File: content/en/docs/_index.md [hunk 9]
```diff
@@ -29,5 +29,57 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+
+### Works similar to package Managers
+
+Operator developers may need to use one or more APIs for the operator installation and OLM helps solve this use case by verifying that all of the required APIs are available for the operator. OLM will attempt to find an operator that owns those missing requiredAPIs and install it. If there is no operator satisfying requiredAPIs, then OLM won't install the main operator.
```
Comment (timflannagan) [inline content/en/docs/_index.md:45]:
```suggestion
Operator developers may need to use one or more APIs for the operator installation and OLM helps solve this use case by verifying that all of the required APIs are available for the operator. OLM will attempt to find and install an operator that owns those missing required APIs. If there are no operators that satisfy the required APIs, then OLM won't install the main operator.
```

File: content/en/docs/_index.md [hunk 10]
```diff
@@ -29,5 +29,57 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+
+### Operator Groups
+
+To give cluster admins complete control over the permissions that OLM grants operators, OLM introduced the concept of `OperatorGroups`.
+
+* Defines the set of permissions that OLM may grant to member operators
+* Defines the set of namespaces that OLM may grant namespaced permissions in.
```
Comment (timflannagan) [inline content/en/docs/_index.md:52]:
```suggestion
* Defines the set of permissions that OLM may grant to member operators
* Defines the set of namespaces that OLM may grant namespaced permissions in
```

File: content/en/docs/_index.md [hunk 11]
```diff
@@ -29,5 +29,57 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+
+## Building blocks of OLM
+
+### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)
+A `ClusterServiceVersion (CSV)` represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc.
```
Comment (timflannagan) [inline content/en/docs/_index.md:57]: Minor nit: it's typically a markdown violation not having a newline between a header and the body.

```suggestion
### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)

A `ClusterServiceVersion (CSV)` represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. 
```

Comment (timflannagan) [reply content/en/docs/_index.md:57]: We may also need to reword: `A `ClusterServiceVersion (CSV)` represents a particular version of a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc.` We may need to flush out this description, but I need to double-check if we already have documentation around these OLM custom resources.

File: content/en/docs/_index.md [hunk 12]
```diff
@@ -29,5 +29,57 @@ Operators must claim ownership of their APIs. OLM will prevent conflicting Opera
...
+## Building blocks of OLM
+
+### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)
+A `ClusterServiceVersion (CSV)` represents a particular version a running operator on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc. 
+
+
+### [CatalogSource](/docs/concepts/crds/catalogsource/)
+A `CatalogSource` represents a store of operator bundles and metadata that OLM can query to discover and install operators and their dependencies.
+
+
```
Comment (timflannagan) [inline content/en/docs/_index.md:63]:
```suggestion
### [CatalogSource](/docs/concepts/crds/catalogsource/)

A `CatalogSource` represents a store of operator bundles and metadata that OLM can query to discover and install operators and their dependencies.

```

File: content/en/docs/_index.md (hunk 11 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md:57]: Sure @timflannagan - I will make newline between a header and body not only in above paragraph but also in rest of the newly added content.

File: content/en/docs/_index.md (hunk 6 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified according to suggestion.

File: content/en/docs/_index.md (hunk 7 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified according to suggestion.

File: content/en/docs/_index.md (hunk 8 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified according to suggestion.

File: content/en/docs/_index.md (hunk 9 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified according to suggestion.

File: content/en/docs/_index.md (hunk 10 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified according to suggestion.

File: content/en/docs/_index.md (hunk 11 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified according to suggestion. Please let me know about the reword of CSV defination.

File: content/en/docs/_index.md (hunk 12 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified according to suggestion.

File: content/en/docs/_index.md [hunk 13]
```diff
@@ -15,19 +15,82 @@ Read more in the [introduction blog post](https://operatorhub.io/what-is-an-oper
...
+* Defines the set of permissions that OLM may grant to member operators
+* Defines the set of namespaces that OLM may grant namespaced permissions in
+
+## Building blocks of OLM
+
+### [ClusterServiceVersion](/docs/concepts/crds/clusterserviceversion)
+
+A `ClusterServiceVersion (CSV)` represents a particular operator version that is running on a cluster. It includes metadata such as name, description, version, repository link, labels, icon, etc.
+
+
```
Comment (timflannagan) [inline content/en/docs/_index.md:65]:
```suggestion

```

File: content/en/docs/_index.md [hunk 14]
```diff
@@ -15,19 +15,82 @@ Read more in the [introduction blog post](https://operatorhub.io/what-is-an-oper
...
+### [CatalogSource](/docs/concepts/crds/catalogsource/)
+
+A `CatalogSource` represents a store of operator bundles and metadata that OLM can query to discover and install operators and their dependencies.
+
+
```
Comment (timflannagan) [inline content/en/docs/_index.md:70]:
```suggestion

```

File: content/en/docs/_index.md [hunk 15]
```diff
@@ -15,19 +15,82 @@ Read more in the [introduction blog post](https://operatorhub.io/what-is-an-oper
...
+### [OperatorCondition](/docs/concepts/crds/operatorcondition/)
+
+An `OperatorCondition` is a CustomResourceDefinition that creates a communication between OLM and an operator it manages. Operators may write to the `Status.Conditions` array to modify the OLM operator management.
+
+### [Subscriptions](/docs/concepts/crds/subscription)
+
+A `Subscription` represents an intention to install an operator. Subscriptions are Custom Resources that relate an operator to a CatalogSource. Subscriptions describe which [channel](/docs/glossary/#channel) of an operator package to subscribe to and whether to perform updates automatically or manually. If set to automatic the Subscription ensures OLM will manage and upgrade the operator to ensure the latest version is always running in the cluster.
+
+
```
Comment (timflannagan) [inline content/en/docs/_index.md:79]:
```suggestion

```

File: content/en/docs/_index.md [hunk 16]
```diff
@@ -15,19 +15,82 @@ Read more in the [introduction blog post](https://operatorhub.io/what-is-an-oper
...
+A `CatalogSource` represents a store of operator bundles and metadata that OLM can query to discover and install operators and their dependencies.
+
+
+### [OperatorCondition](/docs/concepts/crds/operatorcondition/)
+
+An `OperatorCondition` is a CustomResourceDefinition that creates a communication between OLM and an operator it manages. Operators may write to the `Status.Conditions` array to modify the OLM operator management.
+
+### [Subscriptions](/docs/concepts/crds/subscription)
+
+A `Subscription` represents an intention to install an operator. Subscriptions are Custom Resources that relate an operator to a CatalogSource. Subscriptions describe which [channel](/docs/glossary/#channel) of an operator package to subscribe to and whether to perform updates automatically or manually. If set to automatic the Subscription ensures OLM will manage and upgrade the operator to ensure the latest version is always running in the cluster.
```
Comment (timflannagan) [inline content/en/docs/_index.md:77]:
```suggestion
A `Subscription` represents an intention to install an operator. Subscriptions are Custom Resources that relate an operator to a CatalogSource. Subscriptions describe which [channel](/docs/glossary/#channel) of an operator package to subscribe to and whether to perform updates automatically or manually. If set to automatic, the Subscription ensures that OLM will manage and upgrade the operator, verifying the latest version is always running in the cluster.
```

File: content/en/docs/_index.md [hunk 17]
```diff
@@ -15,19 +15,82 @@ Read more in the [introduction blog post](https://operatorhub.io/what-is-an-oper
...
+
+
+### [InstallPlan](/docs/concepts/crds/installplan/)
+
+An `InstallPlan` defines a set of resources to be created in order to install or upgrade to a specific version of an operator.
+
+
```
Comment (timflannagan) [inline content/en/docs/_index.md:84]:
```suggestion

```

File: content/en/docs/_index.md [hunk 18]
```diff
@@ -15,19 +15,82 @@ Read more in the [introduction blog post](https://operatorhub.io/what-is-an-oper
...
+### [OperatorGroup](/docs/concepts/crds/operatorgroup/)
+
+An `OperatorGroup` is an OLM resource that provides rudimentary multitenant configuration to OLM installed operators.
+
+
```
Comment (timflannagan) [inline content/en/docs/_index.md:89]:
```suggestion

```

**Comment by timflannagan** [comment]
Just had a couple more minor nits. This LGTM to otherwise.

File: content/en/docs/_index.md (hunk 13 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: removed extra line.

File: content/en/docs/_index.md (hunk 15 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: removed extra line.

File: content/en/docs/_index.md (hunk 17 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: removed extra line.

File: content/en/docs/_index.md (hunk 18 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: removed extra line.

File: content/en/docs/_index.md (hunk 16 above)
Comment (laxmikantbpandhare) [reply content/en/docs/_index.md]: Modified the description.

**Comment by timflannagan** [comment]
The organization of the introduction page seems a bit strange to me looking at the following screenshot, but nothing that we can't iterate on at a later date:

Screenshot from 2021-03-23 12-50-39

/lgtm

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Review by dinhxuanvu — APPROVED** [review]
/lgtm

**Pull request merged**